src/thx/LocalDate.hx

```diff
@@ -511,6 +496,10 @@ Returns true if this date and the `other` date share the same year and month.
 
   inline public function toDate() : Date
     return new Date(year, month - 1, day, 0, 0, 0);
```

Comment:
I am pretty sure that the decision of having standard `Date` starting at zero was really a bad move and counter intuitive. Ideally you will only use the `thx` date types and you will not have to worry about that. Everything from serialization to parsing uses the `1` based sequence but not the constructor or the property ... that is really messed up. Why are days or years different?